tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -175,30 +176,27 @@ def loadbalancer_setup(
 
 @pytest.mark.e2e
 @pytest.mark.tier1
-def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, rhel7_contenthost):
-    """Register the client using ak to the haproxy
+def test_loadbalancer_install_package(loadbalancer_setup, rhel7_contenthost):
+    r"""Install packages on a content host regardless of the registered capsule being available
 
     :id: bd3c2e50-18e2-4be7-8a7f-c32472e17c61
 
     :Steps:
         1. run `subscription-manager register --org=Your_Organization \
-            --activationkey=Your_Activation_Key \
-            --serverurl=https://loadbalancer.example.com:8443/rhsm \
-            --baseurl=https://loadbalancer.example.com/pulp/content`
-        2. Check which capsule the host got registered.
-        3. Try package installation
-        4. Remove the package and unregister the host
-        5. Again register, verify it's the other capsule this time.
+            --activationkey=Your_Activation_Key \`
+        2. Try package installation
+        3. Check which capsule the host got registered.
+        4. Remove the package
+        5. Take down the capsule that the host was registered to
         6. Try package installation again
 
     :expectedresults: The client should be get the package irrespective of the capsule
         registration.
 
     :CaseLevel: Integration
     """
+    # Register content host
     url = f'https://{loadbalancer_setup["setup_haproxy"]["haproxy"].hostname}'
-    server_url = f'{url}:8443/rhsm'
-    base_url = f'{url}/pulp/content'
 
     result = rhel7_contenthost.download_install_rpm(
```

Comment:
Non-blocking: since you're refactoring this test, consider replacing installing the `katello-ca-consumer` package and registering with sub-man with registering via global registration.